src/components/plan-your-visit/plan-your-visit.stories.js

```diff
@@ -0,0 +1,82 @@
+import html from "../../../.storybook/helpers/html";
+import { withKnobs, text} from "@storybook/addon-knobs";
+
+import { passIcon, dateIcon, locationIcon } from "./helper-icons.js";
+
+import image712 from "../../../.storybook/assets/images/misc/150HubBannerMobile_712w.jpg";
+import image1231 from "../../../.storybook/assets/images/misc/150HubBannerMobile_1231w.jpg";
+
+
+export default {
+	title: "Exhibition/Plan Your Visit",
+	decorators: [withKnobs],
+};
+
+const data = {
```

Comment:
i think by storing the result of `text()` in this data variable, Storybook doesn't know how to render the knobs and watch for changes inside the story.   Ie, `text()` needs to get called dynamically inside your story(ies).  To do that but still make a clean wrapper for the data, I just convert it to a function that returns the object:

ie

```
const data = () => {
header: text()...
}
```

then the story just calls `data().header` instead of `data.header`